### ISSUE TITLE
Impossible to switch account on OSX Mojave

### ISSUE THREAD

**Issue by gufertum** [haiwen/seafile-client#1141]
It's impossible to switch the account on OSX Mojave.
Clicking on the icon (upper left) shows the accounts, but the menu disappears immediately as I move the mouse. Also I cannot get a keyboard focus on it...

This prevents me from switching to the alternative account - completely.
